tests/e2e/test_api.py

```diff
@@ -98,6 +98,7 @@ def fixture_postgres_connection():
 
 
 @pytest.mark.smoketest
+@pytest.mark.introspection
```

Comment:
Why are test_readiness, test_liveness and test_valid_question marked as introspection?

Reply:
Make a note to remove the tags after OLS-1404 then. Anything else blocking merge?

Reply:
Updated decription in OLS-1404